.github/workflows/validation.yaml

```diff
@@ -111,12 +111,12 @@ jobs:
                 const contents = require(`${process.env.GITHUB_WORKSPACE}/${c}`);
 
                 var recipes = []
-                if (contents.instrumentations.resources.length > 0 &&
-                    contents.instrumentations.resources[0].params !== null &&
-                    contents.instrumentations.resources[0].params.recipe_content_url !== null)
-                {
-                  // Look at current recipe_content_url. If it contains any of recipeFiles, add this to the current accumulator
-                  recipes = contents.instrumentations.resources[0].params.recipe_content_url;
+                if (contents.instrumentations.resources) {
+                  contents.instrumentations.resources.forEach(resource => {
+                    if (resource.params && resource.params.recipe_content_url) {
+                      recipes = recipes.concat(resource.params.recipe_content_url);
+                    }
+                  });
```

Comment:
Ah, good catch!

Reply:
Yeah I bumped into that a few times, so I thought it would be great to just fix.